extension/src/experiments/columns/collect/timestamp.ts

```diff
@@ -0,0 +1,14 @@
+import { ColumnAccumulator } from './util'
+import { ColumnType } from '../../webview/contract'
+
+export const timestampColumn = {
+  hasChildren: false,
+  label: 'Timestamp',
+  parentPath: ColumnType.TIMESTAMP,
```

Comment:
[Q] Did you get a chance to look at making this optional? Having this field for the timestamp column does not really make sense.

Reply:
When I attempted it, the timestamp disappeared entirely. Would need to debug further to figure out the issue 🤔

Reply:
You'd need to update this method:

```
  public filterChildren(path?: string) {
    return this.data.filter(element =>
      path
        ? element.parentPath === path
        : Object.values<string>(ColumnType).includes(element.parentPath)
    )
  }

```

in `extension/src/experiments/columns/model.ts`.

The entry point to find that out would be `getChildren` in or around `extension/src/experiments/columns/tree.ts`.